inc/auth.class.php

```diff
@@ -261,7 +261,7 @@ public static function set_user_token($user_login, $current_user)
         /* Send token */
         wp_mail(
             $user_email,
-            __('Your security code', '2-Step-Verification'),
+            esc_html__('Your security code', '2-Step-Verification'),
```

Comment:
Also here a small hint. The WordPress codex like the space after, before brackets, like `esc_html__( 'Your security code', '2-Step-Verification' ),` Also a PHP CodeSniffer topic. But the plugin is currently far away from the codex, so that is not a problem, topic.